auction-server/src/state.rs

```diff
@@ -553,6 +582,7 @@ impl ChainStoreSvm {
                 .parse::<usize>()
                 .expect("Failed to parse router account position"),
         };
+        let (tx, rx) = broadcast::channel(1000);
```

Comment:
I think this doesn't work. If you don't receive the messages in _dummy_log_receiver  they will stay in the channel forever.

`When a value is sent, all [Receiver] handles are notified and will receive the value. The value is stored once inside the channel and cloned on demand for each receiver. Once all receivers have received a clone of the value, the value is released from the channel.`